### DISCUSSION TITLE
osu!lazer thinks that my osu!stable build is some weird one in a remote folder that I used a long time ago. How to make it use my real osu?

### DISCUSSION THREAD

**Discussion by oijdfdg** [ppy/osu#13566] [Q&A] [answered]
osu!lazer thinks that my osu!stable build is some weird one in a remote folder that I used a long time ago. How to make it use my real osu!stable folder?

**Comment by Susko3** [comment, marked as answer]
If you're on Windows, open the Registry Editor, and then delete/modifly the following registry key:

```
HKEY_CLASSES_ROOT\osu\shell\open\command
```

For reference, it's `(Default)` value is `"C:\Users\User\AppData\Local\osu!\osu!.exe" "%1"` for me.

**Reply by peppy** [reply]
Alternative, if you just delete the folder it is attempting to import it will either find your other one or prompt you to locate it.
